models/src_reddit_ads.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 
 sources:
   - name: reddit_ads
-    schema: "{{ var('reddit_ads_schema') }}"
+    schema: "{{ var('reddit_ads_schema') }}" 
```

Comment:
We don't provide a default schema name. If the schema name is not provided then this will fail. Can you make the default schema name for this var be `reddit_ads`.

Reply:
updated